doc/api/process.md

```diff
@@ -452,7 +452,8 @@ added: v0.1.27
 
 The `process.argv` property returns an array containing the command line
 arguments passed when the Node.js process was launched. The first element will
-be [`process.execPath`]. The second element will be the path to the
+be [`process.execPath`]. See `process.argv0` if you need access to the original
+value of `argv[0]` passed to node.  The second element will be the path to the
```

Comment:
In general, please avoid the use of `you` in documentation. For instance, this can be reworded as:

```
See `process.argv0` if access to the original value of `argv[0]` is needed.
```